app/views/navbar.scala.html

```diff
@@ -71,49 +71,49 @@
+                    <a class="navbar-button" id="navbar-guide-btn" href="@routes.ApplicationController.labelingGuide">@Messages("navbar.howto")</a>
                 </li>
                 @if(url.isDefined && url.get == "/audit") {
                     <li class = "active navbar-lnk">
-                        <a class="navbar-button" href='@routes.ApplicationController.help' id="toolbar-help-link" target="_blank">@Messages("navbar.help")</a>
+                        <a class="navbar-button" id="navbar-help-btn" href='@routes.ApplicationController.help' target="_blank">@Messages("navbar.help")</a>
                     </li>
                 }
                 <li class="active dropdown navbar-lnk" id="navbar-data-dropdown-list">
-                    <a id="nav-user-dropdown" class="navbar-button" role="button" data-toggle="dropdown" href="#">
+                    <a id="nav-data-dropdown" class="navbar-button" role="button" data-toggle="dropdown" href="#">
                         @Messages("navbar.data")
                         <b class="caret"></b>
                     </a>
-                    <ul id="nav-user-menu" class="dropdown-menu" role="menu" aria-labelledby="User menu">
+                    <ul id="nav-data-menu" class="dropdown-menu" role="menu" aria-labelledby="User menu">
                         <li>
-                            <a href="@routes.ApplicationController.results">@Messages("navbar.results")</a>
+                            <a id="navbar-results-btn" href="@routes.ApplicationController.results">@Messages("navbar.results")</a>
                         </li>
                         <!-- Only show Label Map link if the city does not have too much data to load that page quickly. -->
                         @if(!List("seattle-wa").contains(Play.configuration.getString("city-id").get)) {
                             <li>
-                                <a href="@routes.ApplicationController.labelMap">@Messages("navbar.labelmap")</a>
+                                <a id="navbar-labelmap-btn"href="@routes.ApplicationController.labelMap">@Messages("navbar.labelmap")</a>
                             </li>
                         }
                         <li>
-                            <a href='@routes.ApplicationController.leaderboard' role="menuitem">
+                            <a id="navbar-leaderboard-btn" href='@routes.ApplicationController.leaderboard' role="menuitem">
                             @Messages("navbar.leaderboard")
                             </a>
                         </li>
                         <li>
-                            <a href='@routes.ApplicationController.developer' id="developer-api-link">@Messages("footer.api")</a>
+                            <a id="developer-api-link-btn" href='@routes.ApplicationController.developer'>@Messages("footer.api")</a>
```

Comment:
Probably want to rename this to fit the pattern of the other buttons (`navbar-api-btn`)